server/pkg/accounts/accounts.go

```diff
@@ -3,14 +3,39 @@ package accounts
 import (
 	"fmt"
 	"os"
+	"errors"
 
 	"github.com/darwinfroese/hacksite/server/models"
 	"github.com/darwinfroese/hacksite/server/pkg/auth"
 	"github.com/darwinfroese/hacksite/server/pkg/database"
 )
 
+const (
+	// UnathorizedErrorMessage is used to determine if the error is because of unathorized account
+	UnathorizedErrorMessage = "Account is invalid"
+)
+
 // CreateAccount will create an account and insert it into the database
 func CreateAccount(db database.Database, account *models.Account) error {
+
+	//Check if the username already exists
+	_, invalidUsername := db.GetAccount(account.Username)
+	if invalidUsername != nil {
```

Comment:
Can you flip this logic so that the code is cleaner? 

`if invalidUsername == nil` and the same for the email check.

This way there are no empty blocks.